examples/django/docker-compose.yml

```
@@ -0,0 +1,24 @@
+services:
+  django:
+    build:
+      context: django-project
+    ports:
+      - "8000:8000"
```

Would it make sense to make the port configurable, in case it conflicts with other projects one might be working on?

```suggestion
      - "${PORT:-8000}:8000"
```

And if `.env` is added to `.gitignore` it can be configured there instead of having to `PORT=9000 docker compose up`.

---

Yes, I like it!